frontend/src/components/VSelectField/VSelectField.vue

```diff
@@ -71,32 +74,58 @@ const splitAttrs = computed(() => {
 </script>
 
 <template>
-  <div
-    class="relative m-0.5px box-content block w-fit rounded-sm border border-default text-sm focus-within:m-0 focus-within:border-1.5 focus-within:border-focus hover:border-hover focus-within:hover:border-focus"
-    :class="splitAttrs.classAttrs"
-  >
-    <div class="pointer-events-none absolute inset-y-0 start-2 my-auto h-fit">
-      <slot name="start" />
-    </div>
-    <div class="pointer-events-none absolute inset-y-0 end-2 my-auto h-fit">
-      <VIcon name="caret-down" />
-    </div>
-    <select
-      :id="fieldId"
-      v-model="selectValue"
-      class="flex h-[calc(theme(spacing.10)_-_2_*_theme(borderWidth.DEFAULT))] appearance-none truncate bg-tx pe-10"
+  <div class="relative m-0.5px">
+    <div
+      v-if="showNewHighlight"
+      class="new-highlight absolute -inset-1.5px animate-new-highlight rounded-[calc(0.125rem_+_1.5px)]"
+      aria-hidden="true"
+    ></div>
+    <div
+      class="relative box-content block w-fit rounded-sm border bg-curr-page text-sm focus-within:m-0 focus-within:border-1.5 focus-within:border-focus hover:border-hover focus-within:hover:border-focus"
       :class="[
-        showSelected ? 'w-full' : 'w-0 max-w-0',
-        hasStartContent ? 'ps-10' : 'ps-2',
+        splitAttrs.classAttrs,
```

Comment:
At first I thought that the snapshot tests were finicky, but they actually did catch a problem. previously, we were binding the `class` attributes to the outer-most div. Now, we are binding them to the inner div instead. However, we have two kind of classes added to `VLanguageSelect`: the one related to the element width (`max-w-full`) and one related to the border (`border-secondary`), which need to be passed to two different elements now (first - to the outer element, and the second - to this div).

I tried moving `max-w-full` from the `VLanguageSelect` in `VFooter` to the `VSelectField`s outer `div`, and it seemed to work fine. But maybe there's a better solution?

Reply:
I switched up the approach in `VSelectField` to add a new element inside the existing `<div>` instead of adding a wrapper `<div>` around it. Hopefully that will avoid the bug you have pointed above.